src/styles/blocks/calendar.scss

```
@@ -0,0 +1,40 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: $calendar-max-width;
+  padding: 10px;
+  &__day {
+    display: flex;
+    flex-basis: 100px;
+    justify-content: center;
+    align-items: center;
+    height: 100px;
+    background-color: $day-cell-bg-color;
+    border: 1px solid $day-cell-border-color;
+    cursor: pointer;
+    transform: 0.5s, background-color 0.5s;
```

Now transitions should work correctly

```suggestion
    transition: transform 0.5s, background-color 0.5s;
```